pkg/vault/vault_impl.go

```diff
@@ -98,7 +98,8 @@ func (c *Connection) RemovePolicyFromIdentity(identity, policyName, auth string)
 // WritePolicy stores in vault the policy indicated.  This can be associated with a vault token or
 // an authentication identity to ensure proper use of secrets.
 // Example policy: "path \"identities/test-identity\" {\n	capabilities = [\"read\"]\n }"
-// 		NOTE the line returns and the tab.  Without them it fails!
+//
```

Comment:
Remove the empty line

Reply:
again, I tried to separate Note with the empty line